Dockerfile

```diff
@@ -1,36 +1,24 @@
-# fog-sw BUILDER
-ARG FROM_IMAGE
-FROM $FROM_IMAGE as fog-sw-builder
-ARG ROS_DISTRO="galactic"
-ARG UID=1000
-ARG GID=1000
-ARG PACKAGE_NAME
-
-WORKDIR /$PACKAGE_NAME/main_ws
-USER root
-ADD . /$PACKAGE_NAME/main_ws/src
-RUN chown -R builder:builder /$PACKAGE_NAME/main_ws
-
-USER builder
-
-RUN if [ -e /$PACKAGE_NAME/deps_ws ]; then \
-        . /$PACKAGE_NAME/deps_ws/install/setup.sh && \
-        colcon build; \
-    elif [ -e /opt/ros/${ROS_DISTRO}/setup.sh ]; then \
-        . /opt/ros/${ROS_DISTRO}/setup.sh && \
-        colcon build; \
-    fi 
-
-RUN sed --in-place \
-      's|^source .*|source "/'$PACKAGE_NAME'/main_ws/install/setup.bash"|' \
-      /$PACKAGE_NAME/entrypoint.sh && \  
-        chmod +x /$PACKAGE_NAME/entrypoint.sh
-
-ENV PACKAGE_NAME $PACKAGE_NAME
-ENV RMW_IMPLEMENTATION rmw_fastrtps_cpp
-#ENV RMW_IMPLEMENTATION rmw_cyclonedds_cpp
-
-ENV FASTRTPS_DEFAULT_PROFILES_FILE /$PACKAGE_NAME/main_ws/src/DEFAULT_FASTRTPS_PROFILES.xml
-
-WORKDIR /$PACKAGE_NAME
-ENTRYPOINT "/"$PACKAGE_NAME"/entrypoint.sh"
+FROM ghcr.io/tiiuae/fog-ros-baseimage:builder-latest AS builder
+
+COPY . /main_ws/src/
+
+# this:
+# 1) builds the application
+# 2) packages the application as .deb & writes it to /main_ws/
+RUN /packaging/build.sh
+
+#  ▲               runtime ──┐
+#  └── build                 ▼
+
+FROM ghcr.io/tiiuae/fog-ros-baseimage:sha-7857f7d
```

Comment:
Should we use newer version of the image? -> 
https://github.com/tiiuae/fog-ros-baseimage/pkgs/container/fog-ros-baseimage/18451119?tag=sha-1cabd43

Reply:
Good catch, yeah I think it'd be safest if all projects used the exact same version, and the version you pointed out is newer and has some important updates!

I think we should make soon a script that pulls `Dockerfile` from each project and print out which base image version it uses, and we should check that script before each version release

Reply:
After our team brainstorming in slack. Should we update the dockerfiles with stable tag?

Reply:
Yes. Done. Thanks again!